emails/src/partials/project-team.html

```diff
@@ -63,15 +63,15 @@
                         {{#if [notifications.connect.project.member.invite.requested]}}
                             You are requested to add <strong>{{userFullName}}</strong> as a copilot
                         {{/if}}
-                        {{#if [notifications.connect.project.member.copilot.added]}}
+                        {{#if [notifications.connect.project.member.invite.approved]}}
                             {{#if [creator]}}
```

Comment:
@gondzo I think we need to rename this creator as well, need not we ?

Reply:
this is done already @RishiRajSahu